kanidm_tools/src/cli/account.rs

```diff
@@ -68,7 +72,13 @@ impl AccountOpt {
                         acsopt.aopts.account_id.as_str(),
                         password.as_str(),
                     ) {
-                        eprintln!("Error -> {:?}", e);
+                        match e {
+                            // TODO: once the password length is configurable at a system level (#498), pull from the configuration.
+                            ClientErrorHttp(_, Some(PasswordBadListed), _) => error!("Password is banned by the administrator of this system, please try a different one."),
+                            ClientErrorHttp(_, Some(PasswordTooShort(PW_MIN_LENGTH)), _) => error!("Password was too short (needs to be at least {} characters), please try again.", PW_MIN_LENGTH),
+                            ClientErrorHttp(_, Some(PasswordTooWeak), _) => error!("Password too weak, please try a more complex password."),
```

Comment:
I think it could be good to open a subsequent issue to have better errors returned around password qual in general.

Reply:
I've updated the wording a little, and yep, new issue created.